launcher/ui/pages/modplatform/import_ftb/ListModel.cpp

```diff
@@ -41,27 +43,54 @@ QString getStaticPath()
 #else
     partialPath = QDir::homePath();
 #endif
-    return FS::PathCombine(partialPath, ".ftba");
+    return FS::PathCombine(partialPath, ".ftba", "instances");
 }
 
-static const QString FTB_APP_PATH = FS::PathCombine(getStaticPath(), "instances");
+QString getDynamicPath()
+{
+    auto settingsPath = FS::PathCombine(QDir::homePath(), ".ftba", "bin", "settings.json");
```

Comment:
This will only work on Linux (I'm fairly sure)
```suggestion
    auto settingsPath = FS::PathCombine(getFTBRoot(), "bin", "settings.json");
```
Maybe make a new function

Reply:
Do I get to reuse it anywhere? or is it a function for code clarity?

Reply:
You're already doing the OS check in getStaticPath so you might as well extract it instead of duplicating it

Reply:
This is not the same: from the code for both windows and linux the ftb settings are in "<home>/.ftba/bin/settings.json" The staticpath  gets the default value of the instance folders (and this one points towards a different path on windows) 
So the static path doesn't point inside the ftb root directory on windows.

Reply:
That is to say, on Windows, the default path for instances is `C:\Users\Me\AppData\Roaming\.ftba\instances`, but the path for settings is `C:\Users\Me\.ftba\bin\settings.json`?
Are you sure you're not mistaken?

Reply:
https://github.com/FTBTeam/FTB-App/blob/9dc79361646e18231d1764acc385996f5f1726f3/src/background.ts#L37
and the getAppHome is defined here: https://github.com/FTBTeam/FTB-App/blob/9dc79361646e18231d1764acc385996f5f1726f3/src/background.ts#L14-L20

But now that I added the dynamic path I think I will remove the static one and keep only the userDefined setting and the dynamic found one

Reply:
https://github.com/FTBTeam/FTB-App/blob/9dc79361646e18231d1764acc385996f5f1726f3/src/background.ts#L15
It's not the same on macos

Reply:
Yeah I saw that and updated the code